packages/protocol/scripts/truffle/make-release.ts

```diff
@@ -210,7 +223,9 @@ module.exports = async (callback: (error?: any) => number) => {
       }
     }
     for (const contractName of contracts) {
-      await release(contractName)
+      if (await isCoreContract(contractName)) {
```

Comment:
Will this not skip all linked libraries?

Reply:
I believe each `release` will deploy the linked library separately